app/views/granules/prepare_script.html.erb

```diff
@@ -191,9 +196,9 @@ fetch_urls <<'EDSCEOF'
       var links = doc.links;
       preContent = document.getElementById('script');
       preContent.innerHTML += links.join("\n");
-      preContent.innerHTML += '\n'
       var next = getNextPageUrl();
       if (next) {
+        preContent.innerHTML += '\n'
```

Comment:
Now it only adds the carriage return if there *is* a 'next' - if there isn't, then it doesn't add a line.  This stops it from adding that extra carriage return at the end of the script file.